style_guide.md

```diff
@@ -3,5 +3,5 @@
 * No contractions except don't, can't and doesn't
 * Capitalize the first letter and proper nouns in titles
 * Capitalize branded software (e.g., Excel)
-* To abbreviate `for example` use `e.g.,`
-* Always spell check
\ No newline at end of file
+* To abbreviate `for example` use `e.g.,` << How do you feel about `in other words` or `that is` converted to `i.e.`
```

Comment:
I'm for it, as long as it's "i.e.," (i.e., with the periods and the comma).

Reply:
I agree. As context, most people agree that `e.g.` should have a comma, but ppl are divided on `i.e.`. AP says use comma after both so lets do that.

Here is some fun reading: https://www.dailywritingtips.com/comma-after-i-e-and-e-g/